source/deployment/cluster.rst

```diff
@@ -134,75 +134,143 @@ A sample configuration for NGINX is provided below.  It assumes you have two Mat
 
-Once loaded, database write requests will be sent to the master database and read requests will be distributed among the other databases in the list.
+Once loaded, database write requests are sent to the master database and read requests are distributed among the other databases in the list.
 
 Loading a multi-database configuration onto an active server
-``````````````````````````````````````````````````````````````````````````````````````````````````
-After a multi-database configuration has been defined in ``config.json`` the following procedure can be used to apply the settings without shutting down the Mattermost server: 
+````````````````````````````````````````````````````````````
 
-1. Go to **System Console** > **Configuration** and press **Reload Configuration from Disk** to reload configuration settings for the Mattermost server from ``config.json``. 
-2. Go to **System Console** > **Database** and press **Recycle Database Connections** to takedown existing database connections and set up new connections in the multi-database configuration. 
+After a multi-database configuration has been defined in ``config.json``, the following procedure can be used to apply the settings without shutting down the Mattermost server: 
 
-While connection settings are changing there may be a brief moment when writes to the master database will be unsuccessful. The process waits for all existing connections to finish and starts serving new requests with the new connections. End users attempting to send messages while the switch is happening will have an experience similar to losing connection to the Mattermost server.
+1. Go to **System Console > Configuration** and click **Reload Configuration from Disk** to reload configuration settings for the Mattermost server from ``config.json``. 
+2. Go to **System Console > Database** and click **Recycle Database Connections** to take down existing database connections and set up new connections in the multi-database configuration.
 
-Manual failover for master database  
-`````````````````````````````````````````````````````````````````
-If the need arises to switch from the current master database--for example, if it is running out of disk space, or requires maintenance updates, or for other reasons--the Mattermost server can switch to using one of its read replicas as a master database by updating ``DataSource`` in ``config.json``. The following procedure can be used to apply the settings without shutting down the Mattermost server: 
+While the connection settings are changing, there might be a brief moment when writes to the master database are unsuccessful. The process waits for all existing connections to finish and starts serving new requests with the new connections. End users attempting to send messages while the switch is happening will have an experience similar to losing connection to the Mattermost server.
```

Comment:
Huh. I made the change, but for some reason it's not showing up here. You can see it in commit https://github.com/mattermost/docs/pull/825/commits/9504dfa7a9658e6a540f7aab81865078c7e782a7, line 205.